_events/6.markdown

```diff
@@ -1,10 +1,10 @@
 ---
 layout: event
 title:  "2019-12-03 Call"
-date:   2019-12-03 9:00
+date:   2019-12-03 10:00
 file_list:
-  # google drive links
-summary: Discussion of meta data use cases.
+- https://docs.google.com/presentation/d/1JZtv_q-fgXBoi1efw-VSJvQYCBEPx-SwehaRnLCQLTM/edit?usp=sharing
```

Comment:
We don't need the dash -, just the indent like in https://raw.githubusercontent.com/hms-dbmi/x-consortium/master/_example_templates/0101.markdown

Reply:
@mvella1 i think this is still here

Reply:
i don't think the dash is necessary still